app/src/main/java/cm/aptoide/pt/home/ErrorHomeBundle.java

```diff
@@ -0,0 +1,7 @@
+package cm.aptoide.pt.home;
+
+public class ErrorHomeBundle extends DummyBundle {
```

Comment:
This wasn't you and you don't have to do anything about this... But it just seems unnecessary to have a DummyBundle just to avoid implementing null 🤣